web/src/hotkey.js

```diff
@@ -974,9 +975,15 @@ export function process_hotkey(e, hotkey) {
         case "show_sender_info":
             popovers.show_sender_info();
             return true;
-        case "toggle_reactions_popover": // ':': open reactions to message
-            reactions.open_reactions_popover();
+        // ':': open reactions to message
+        case "toggle_reactions_popover": {
+            const $row = message_lists.current.selected_row();
+            emoji_picker.toggle_emoji_popover(
+                msg.sent_by_me ? $row.find(".actions_hover")[0] : $row.find(".reaction_button")[0],
```

Comment:
This might be a bit nicer to just have an if/else that sets `reactions_picker_container` as either `actions_hover` or `reaction_button`... but it might be significantly nicer with a different change, so I'm OK with this.